### ISSUE TITLE
"Full list of bad words and top swear words banned by Google." source?

### ISSUE THREAD

**Issue by Arcanorum** [coffee-and-fun/Google-profanity-words#7]
Where is this claim coming from?

**Comment by RobertJGabriel** [comment]
@Arcanorum 

It was an old Project by Google, its removed now but here is some news sources 

https://thenextweb.com/news/google-inadvertently-creates-a-profanity-api

**Comment by meotimdihia** [comment]
@RobertJGabriel where did you pull this data? we need proof for this :)

**Comment by RobertJGabriel** [comment]
Hey @meotimdihia,

It was taken at the time from api endpoint that was life at the time. This was 2012.

They left it open, I will try and get more details but it was 11 years ago and I'm getting up their in age. But you can see the link above.

Let me know what you need and I will try and get it for you.

**Comment by meotimdihia** [comment]
@RobertJGabriel thanks for the answer, then the data may be outdated because it was 11 years ago.

**Comment by RobertJGabriel** [comment]
@meotimdihia yes but we have added more words from pull requests too. The original list was much smaller.

**Comment by meotimdihia** [comment]
@RobertJGabriel I got it, thanks.

**Comment by RobertJGabriel** [comment]
Your welcome and if need anything else just mention below.
